### DISCUSSION TITLE
Highest compression for AVIF images and google pagespeed

### DISCUSSION THREAD

**Discussion by dgrl** [ImageMagick/ImageMagick#7740] [Help]
### ImageMagick version

latest 

### Operating system, version and so on

Cpanel php

### Description

Hi there

How are you doing?
Hope all is well.

I would like to know how to get the highest compression using Imagick::setCompressionQuality

When I use 1 as compression quality I still get the following error on pagespeed.

Properly size images 

Since I am already using 1 as highest compression I wonder why pagespeed tells me to compress it even more.

Using 0 does not seems to work
Am I missing some other settings?

Awaiting your response.

Kind regards
DG

**Comment by snibgo** [comment]
This is an ImageMagick forum. Your question seems to be about IMagick, which is a different system (an interface between PHP and ImageMagick).

Or perhaps your question is about pagespeed, whatever that is.

**Comment by dgrl** [comment]
Question can be deleted since its in the wrong forum
